sdk/servicelinker/azure-resourcemanager-servicelinker/src/samples/README.md

```diff
@@ -0,0 +1,57 @@
+<!-- ---
+page_type: sample
+languages:
+  - java
+products:
+  - azure
+  - azure-service-linker
+urlFragment: service-linker-resourcemanager-samples
+--- -->
+
+# Azure Service Linker Create Connection Samples for Java
+
+This contains Java samples for create service connection using Service Linker Provider.
+
+## Getting started
+
+Getting started explained in detail [here][SERVICELINKER_README_GETTING_STARTED].
+Please refer it to add dependency and configure authentication environment variables.
+
+## Samples Details
+
+All samples are located in [CreateServiceLinker][SERVICELINKER_SAMPLE_CODE] file, you can use `main()` entrypoint to run it directly.
+
+### CreatSpringCloudAndSQLConnection
+
+1. Create Spring Cloud Deployment
+2. Create SQL Database
+3. Create Service Linker for Deployment and SQL using SQL username and password
```

Comment:
setup connection between spring cloud app and sql database using username and password by creating service linker

Reply:
done